bootstrap/test/nerves_new_test.exs

```diff
@@ -68,4 +68,13 @@ defmodule Nerves.BootstrapTest do
     end
   end
 
+  test "new project default cookie set", context do
+    in_tmp context.test, fn ->
+      Mix.Tasks.Nerves.New.run([@app_name])
+      assert_file "#{@app_name}/rel/vm.args", fn file ->
+        assert file =~ ~r/.+?(?:-setcookie ).{64}\n/s
```

Comment:
If this works, I'm OK with it, but I think you could simplify a few things:
* I think `.+?` is probably the same as `.*`, but I've never seen `+?` used together like that so maybe it does something else.
* I think the `(?:` ... `)` structure is used to specify a non-capturing match group, which is doesn't seem like something you need here.
* It would be more reliable to anchor on `$` as the end-of-line instead of `\n`. For example, on Windows, it'll be `\r\n`, so there will be 65 characters there instead of `64`, but `$` handles that for you.